cobalt/build/build_info.py

```diff
@@ -77,13 +78,27 @@ def _get_subject_from_last_commit(cwd):
   return _get_last_commit_with_format(r'%s', cwd=cwd)
 
 
+def _get_cobalt_version():
+  version_header_path = os.path.join(os.path.dirname(FILE_DIR), 'version.h')
+  contents = ''
+  with open(version_header_path, 'r', encoding='utf-8') as f:
+    contents = f.read()
+  compiled_cobalt_version_pattern = re.compile(
+      _COBALT_VERSION_PATTERN, flags=re.MULTILINE)
```

Comment:
`flags=re.MULTILINE` is not necessary. The pattern you are matching with only matches against a single line

Reply:
removing the flag actually breaks the regex so that it fails to find it

Reply:
Oh, I wonder if the flag is required to match a full line then. I don't mind if it stays.